deptofdefense/move.mil
# Weight estimator content updates (icons, text, data)

## Checklist

I have…


## Summary of Changes

This pull request…

- Update icons + text (https://github.com/deptofdefense/TRANSCOM/issues/28)
- Update data based on Lilly's changes in this Google sheet: https://docs.google.com/spreadsheets/d/1iyRqnq_Zed-sEmvYN_0xWP2V88iIE0BSKkwJ8GZ3CMM/edit#gid=0

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
2. git checkout weight-estimator-updates
3. set up development dependencies according to CONTRIBUTING.md,
4. run bin/rails db:drop db:create db:migrate db:seed,
5. run bin/rails server, and
6. load up http://localhost:3000/resources/weight-estimator in your Web browser of choice, check content

## Screenshots

<img width="805" alt="screen shot 2017-10-25 at 10 35 11 am" src="https://user-images.githubusercontent.com/29409348/32004620-36042542-b970-11e7-90b5-5b2e49a5b1aa.png">
